Sources/Algorithms/Partition.swift

```diff
@@ -210,8 +210,10 @@ extension Collection {
 //===----------------------------------------------------------------------===//
 
 extension Sequence {
-  /// Returns two arrays containing, in order, the elements of the sequence that
-  /// do and don’t satisfy the given predicate.
+  /// Returns two arrays containing the elements of the sequence that
+  /// don’t and do satisfy the given predicate, respectively. The order of the
```

Comment:
The abstract should be a single sentence. Could you move the second sentence down to the `Returns:` section?